src/games/strategy/triplea/printgenerator/UnitInformation.java

```diff
@@ -82,10 +82,8 @@ protected void saveToFile(final PrintGenerationData printData, final Map<UnitTyp
       unitInformation.write("\r\n");
       unitInformation.close();
     } catch (final FileNotFoundException e) {
-      // TODO Auto-generated catch block
       e.printStackTrace();
     } catch (final IOException e) {
-      // TODO Auto-generated catch block
```

Comment:
More bad error handling, missing the error message contained in the exception. With just the stack trace, we will never what or why there was an IOException